dot/network/notifications.go

```diff
@@ -198,6 +198,11 @@ func (s *Service) createNotificationsMessageHandler(info *notificationsProtocol,
 }
 
 func (s *Service) sendData(peer peer.ID, hs Handshake, info *notificationsProtocol, msg NotificationsMessage) {
+	if support, err := s.host.supportsProtocol(peer, info.protocolID); err != nil || !support {
+		logger.Warn("the peer does not supports the protocol", "protocol", info.protocolID, "peer", peer, "err", err)
```

Comment:
This should be set at `debug` level as we may get this error often.

Reply:
Done!